atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/atomic/CassImitatingConsensusForgettingStoreV4.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.atomic;
+
+import com.palantir.atlasdb.keyvalue.api.Cell;
+import com.palantir.atlasdb.keyvalue.api.CheckAndSetException;
+import com.palantir.atlasdb.keyvalue.api.KeyAlreadyExistsException;
+import com.palantir.common.streams.KeyedStream;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public final class CassImitatingConsensusForgettingStoreV4 extends CassandraImitatingConsensusForgettingStore {
+    static final byte[] IN_PROGRESS_MARKER = new byte[1];
```

Comment:
This should either be private or visible for testing I think? If not I'd suggest leaving a comment

Reply:
this is a testing package so thought it was implied. happy to annotate